integration/integration_test.py

```diff
@@ -0,0 +1,48 @@
+"""@package integration_test.py
+Entry point for our integration testing
+"""
+
+import logging
+import os
+import sys
+import unittest
+from integration_test_base import IntegrationTestBase
+
+import test_integration_basics
+
+
+def determineIpVersionsFromEnvironment():
+  ipv4 = True
+  ipv6 = True
+  versions = os.environ.get("ENVOY_IP_TEST_VERSIONS", "all")
+  if versions == "v4only":
+    ipv6 = False
+  elif versions == "v6only":
+    ipv4 = False
+  elif versions == "all":
+    pass
+  else:
+    raise Exception("Unknown ip version: '%s'" % versions)
```

Comment:
Nit: this should be a custom `NighthawkError` or more specific, subclassing `Exception`. This allows you to catch the more specific subclass.

Reply:
Done